src/components/input.js

```diff
@@ -248,8 +248,14 @@ const withAutocomplete = WrappedComponent => ({
     })
   })
 
+  // Unused param denoted as _u
```

Comment:
Is this comment necessary?

Reply:
resolved with a variable rename - thanks!